x-pack/plugins/apm/common/processor_event.ts

```diff
@@ -8,4 +8,12 @@ export enum ProcessorEvent {
   transaction = 'transaction',
   error = 'error',
   metric = 'metric',
+  span = 'span',
+  onboarding = 'onboarding',
+  sourcemap = 'sourcemap',
 }
+
+export type UIProcessorEvent =
```

Comment:
Can we put a JSDoc comment on here explaining what this means and what it's used for?